### PR TITLE
Feat ADE mosaic

### PR DESCRIPTION
**Pull request by wildintellect**
Notebook demonstrating how to wrap ADE data into a mosaicjson and render it through the tile service for in notebook visualization of large datasets.

TODO: probably missing changes to add this to TOC. @abarciauskas-bgse should we keep these out of the TOC, since it's early drafts?

Much of this information will get updated later with work from Grace on pyCMC improvements.

### CODE REVIEW

File: docs/source/visualization/ade_mosaic.ipynb [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to find the s3 path\n",
+    "\n",
+    "You might be wondering how to find the s3 path to begin with. Right now the easiest way is to right click on a file in the file explorer on the left panel, and Get Presigned S3 Url.\n",
+    "\n",
+    "It will look something like `https://maap-ops-dataset.s3.amazonaws.com/maap-users/alexdevseed/landsat8/viz/Landsat8_30542_comp_cog_2015-2020_dps.tif?AWSAccessKeyId...`\n",
+    "\n",
+    "The first part of the url is the bucket name: `maap-ops-dataset`\n",
```
Comment (spa0002) [inline docs/source/visualization/ade_mosaic.ipynb:97]: Appears there is missing period at end of sentence - https://github.com/MAAP-Project/maap-documentation/blob/feat/ADEmosaic/docs/source/visualization/ade_mosaic.ipynb

**Review by spa0002 — COMMENTED** [review]
Note that the execution counts are not in order.
Added review comments

*adding reminder that at some point ade_mosaic.ipynb will need to be added to index.rst in source folder to appear in table of contents.

**Comment by abarciauskas-bgse** [comment]
Thanks for adding this @wildintellect just noting that I get an access denied error trying to create a mosaic in my workspace so it appears my user does not have access to those files `s3://maap-ops-dataset/maap-users/alexdevseed/landsat8/viz/Landsat8_30542_comp_cog_2015-2020_dps.tif` however the pyCMC still works as I would expect since the dynamic tiler is deployed in UAH which should have access to `s3://maap-ops-dataset` (I think)

**Comment by wildintellect** [comment]
@abarciauskas-bgse are you in the Pilot Ops ADE? and is your user/workspace part of the maap-users organization?

File: docs/source/visualization/ade_mosaic.ipynb [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### How to find the s3 path\n",
+    "\n",
+    "You might be wondering how to find the s3 path to begin with. Right now the easiest way is to right click on a file in the file explorer on the left panel, and Get Presigned S3 Url.\n",
+    "\n",
+    "It will look something like `https://maap-ops-dataset.s3.amazonaws.com/maap-users/alexdevseed/landsat8/viz/Landsat8_30542_comp_cog_2015-2020_dps.tif?AWSAccessKeyId...`\n",
```
[the remark this answers is not in the document]
Comment (wildintellect) [reply docs/source/visualization/ade_mosaic.ipynb:95]: Will look into it, not sure how to make it better.

**Pull request merged**